pkg/check/util.go

```diff
@@ -157,3 +159,31 @@ func getAttachedTagsOnObjects(ctx *CheckContext, referencesToCheck []mo.Referenc
 	attachedTags, err := tagManager.GetAttachedTagsOnObjects(tctx, referencesToCheck)
 	return attachedTags, err
 }
+
+// getVirtualMachine returns VirtualMachine based on provider ID passed in.  This will
+// also load all properties related to VM using NodeProperties
+func getVirtualMachine(ctx *CheckContext, dc *object.Datacenter, providerID string) (*mo.VirtualMachine, error) {
```

Comment:
Is this only used in tests? If yes - then it should be part of the test file, unless there is a good reason for keeping it here.

Reply:
Originally I added when I had the cluster-wide check.  I will move this into the test file.